elements/bulbs-video/components/revealed.js

```diff
@@ -156,13 +160,54 @@ export default class Revealed extends React.Component {
     return false;
   }
 
-  vastUrl (videoMeta) {
-    let baseUrl = 'http://us-theonion.videoplaza.tv/proxy/distributor/v2?rt=vast_2.0';
+  getProfValue () {
+    if (window.isMobile.any) {
+      return 'theonion_mobileweb_html5';
+    } else {
+      return 'theonion_desktop_html5';
+    }
+  }
+
+  setDeviceAcronym () {
+    if (window.isMobile.any) {
+      return 'm';
+    } else {
+      return 'd';
+    }
+  }
+
+  getSiteName () {
+    return window.location.host.split('.')[1];
+  }
 
+  getDfpSection () {
+    if (window.TARGETING.dfp_section) {
+      return window.TARGETING.dfp_section;
+    } else if (window.TARGETING.dfp_specialcoverage) {
+      let slug = window.TARGETING.dfp_specialcoverage;
+      return `specialcoverage_${slug}`;
+    } else {
+      return 'video';
+    }
+  }
+
+  getCsidValue (videoMeta) {
+    let deviceAcronym = this.setDeviceAcronym();
```

Comment:
I was trying to differentiate from values we are pulling in from other places the  vs values we are setting.